requirements/base.txt

```diff
@@ -9,18 +9,18 @@ contourpy==1.2.1
     # via matplotlib
 cycler==0.12.1
     # via matplotlib
-fonttools==4.51.0
+fonttools==4.53.0
     # via matplotlib
 kiwisolver==1.4.5
     # via matplotlib
 matplotlib==3.9.0
     # via -r base.in
-numpy==1.26.4
+numpy==2.0.0
```

Comment:
Is `scipp` compatible now with numpy 2.0? I thought we had not yet released that?

Can we make the changes with `np.NINF` etc without updating the dependencies?